docs/src/modules/components/MarkdownElement.js

```diff
@@ -408,6 +408,9 @@ const Root = styled('div')(
   }),
   {
     ':where(.mode-dark) &': {
+      '& :not(pre) > code': {
```

Comment:
Maybe to make it clearer

```suggestion
      // inline code block
      '& :not(pre) > code': {
```